Lib/test/test_asyncio/test_tasks.py

```diff
@@ -2688,6 +2688,28 @@ def test_get_context(self):
         finally:
             loop.close()
 
+    def test_proper_refcounts(self):
+        # see: https://github.com/python/cpython/issues/126083
+        class Break:
+            def __str__(self):
+                raise RuntimeError("break")
+
+        obj = object()
+        initial_refcount = sys.getrefcount(obj)
+
+        coro = coroutine_function()
+        loop = asyncio.new_event_loop()
+        task = asyncio.Task.__new__(asyncio.Task)
+
+        for _ in range(5):
+            with self.assertRaisesRegex(RuntimeError, 'break'):
+                task.__init__(coro, loop=loop, context=obj, name=Break())
+
+        coro.close()
+        del task
+
+        self.assertEqual(sys.getrefcount(obj), initial_refcount)
```

Comment:
Please avoid tests for exact refcounts, the test should test that no references are leaked rather than reference count should be equal, both are not same thing. In this particular case all of this seems unnecessary, out existing refleak checker is sufficient.

Reply:
Oh I was unaware of the existing refleak checker. Do you have any examples of where it's used so i can check it out?